packages/babel-plugin-polyfill-corejs3/test/fixtures/usage-global/all-proposals-chrome-90-corejs-3.16/output.mjs

```diff
@@ -0,0 +1,66 @@
+import "core-js/modules/esnext.array.find-last.js";
+import "core-js/modules/esnext.array.find-last-index.js";
+import "core-js/modules/esnext.async-iterator.constructor.js";
+import "core-js/modules/esnext.async-iterator.to-array.js";
+import "core-js/modules/esnext.iterator.constructor.js";
+import "core-js/modules/esnext.iterator.to-array.js";
+import "core-js/modules/esnext.async-iterator.for-each.js";
+import "core-js/modules/esnext.iterator.for-each.js";
```

Comment:
To avoid problems, modules should be sorted in the order from `core-js-compat/data`. However, seems it's out of this PR scope.